examples/ner_new/ner_model_processor.py

```diff
@@ -0,0 +1,277 @@
+#  Copyright 2020 The Forte Authors. All Rights Reserved.
+#  #
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  #
+#       http://www.apache.org/licenses/LICENSE-2.0
+#  #
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# All the configs
+import logging
+import os
+from pathlib import Path
+import numpy as np
+import torch
+import yaml
+from typing import Dict, Optional, List
+
+from ft.onto.base_ontology import Sentence, EntityMention, Token
+from texar.torch.data import Batch
+from torch.optim import SGD
+from torch import nn
+from torch.optim.optimizer import Optimizer
+
+from forte.data.base_pack import PackType
+from forte.data.extractor.model_processor import ModelProcessor
+from forte.models.ner.model_factory import BiRecurrentConvCRF
+from forte.models.ner.utils import load_glove_embedding
+from forte.data.extractor.extractor import BaseExtractor
+from forte.common.configuration import Config
+
+
+def construct_word_embedding_table(embed_dict, extractor: BaseExtractor):
```

Comment:
I've seen this function a few times, can we make a library util from it?

Reply:
This file is actually deprecated. Sorry for this.